rudder-agent/SOURCES/uuidgen

```diff
@@ -1,6 +1,7 @@
 #!/bin/sh
 
 # uuidgen doesn't exist on AIX, so provide a compatible shell function
+# This is not compatible with Linux systems.
 
```

Comment:
Not sure how we should handle the case where there is a uuidgen binary on the system. 
* When this script is called, it is added to the path, so we would need to check the path of the script and see if there is another available uuidgen...
* We could also change agent-check to stop adding /opt/rudder/bin to the path and choose our uuidgen only if it cannot find one on the system.
* Finally, I don't really like calling it uuidgen as it is not a drop-in replacement (it does not support any options).

We could move all this logic to rudder-agent common lib and have a standard internal uuid generation with platform detection.